<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="INLPF1AX91E" failures="0" tests="2" name="Framework.Eshopping.Potest" time="0.224" errors="1" timestamp="2024-11-24T18:34:29 IST" skipped="1">
  <testcase classname="Framework.Eshopping.Potest" name="placeOrder" time="0.224">
    <error message="
[public void Framework.Eshopping.Potest.placeOrder(java.util.HashMap,java.lang.String) throws java.io.IOException] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: placeOrder([java.util.HashMap&lt;java.lang.String, java.lang.String&gt; arg0, java.lang.String arg1])
Arguments: [(java.util.HashMap) {password=Test@1234, productName=ZARA COAT 3, username=satish@gmail.com}]" type="org.testng.internal.reflect.MethodMatcherException">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void Framework.Eshopping.Potest.placeOrder(java.util.HashMap,java.lang.String) throws java.io.IOException] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: placeOrder([java.util.HashMap<java.lang.String, java.lang.String> arg0, java.lang.String arg1])
Arguments: [(java.util.HashMap) {password=Test@1234, productName=ZARA COAT 3, username=satish@gmail.com}]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:46)
at org.testng.internal.Parameters.injectParameters(Parameters.java:918)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- placeOrder -->
  <testcase classname="Framework.Eshopping.Potest" name="verifyProductDisplayInOrderPage" time="0.000">
    <skipped/>
  </testcase> <!-- verifyProductDisplayInOrderPage -->
</testsuite> <!-- Framework.Eshopping.Potest -->
